docs add/edit docstrings

agirodi opened the pull request:
This PR adds missing and edits existing Docstrings to various files to match [Google Docstring Styleguide](https://google.github.io/styleguide/pyguide.html)  
Files edited so far:
- FastsurferCNN/
  - config/
    - defaults.py
    - global_var.py
  - data_loader/
    - augmentation.py
    - conform.py
    - loader.py
  - models/
    - interpolation_layer.py
    - losses.py
    - networks.py
    - optimizer.py
    - sub_module.py
  - utils/
    - checkpoint.py
    - load_config.py
    - logging.py
    - lr_scheduler.py
    - meters.py
    - metrics.py
    - misc.py
  - generate_hdf5.py
  - inference.py
  - run_model.py
  - run_prediction.py
  - train.py
- recon_surf/
  - utils/
    - extract_recon_surf_time_info.py
  - N4_bias_correct.py
  - align_points.py
  - align_seg.py
  - create_annotation.py
  - fs_balabels.py
  - image_io.py
  - lta.py
  - map_surf_label.py
  - paint_cc_into_pred.py
  - rewrite_mc_surface.py
  - rotate_sphere.py
  - smooth_aparc.py
  - spherically_project.py
  - spherically_project_wrapper.py

agirodi committed:
docs add docstrings

agirodi committed:
docs add docstrings

agirodi committed:
docs add docstrings

agirodi committed:
docs add/edit docstrings

Adding/Reformating Docstrings for important functions in: augmentation.py, defaults.py, extract_recon_surf_time_info.py, global_var.py, run_model.py, run_prediction.py

agirodi committed:
docs add docstrings

Adding Docstrings for augmentation.py

agirodi committed:
docs add docstrings

Adding and editing Docstrings

agirodi committed:
docs add docstrings and typo

typo post_predition_mapping_hook -> post_prediction_mapping_hook

agirodi committed:
docs add docstrings

agirodi committed:
docs add docstrings

agirodi committed:
docs add docstrings

align_points.py

agirodi committed:
docs add docstrings

align_seg.py, create_annotation.py, fs_balabels.py, image_io.py

agirodi committed:
add list

agirodi committed:
docs add docstrings

load_config.py, lr_scheduler.py, meters.py, optimizer.py

agirodi committed:
docs add docstrings

metrics.py, misc.py

agirodi committed:
docs add/edit docstrings

dataset.py, lta.py, map_surf_label.py, meters.py, misc.py, networks.py, run_prediction.py, sub_module.py

agirodi committed:
docs add/edit docstrings

N4_bias_correct.py, map_surf_label.py

agirodi committed:
docs edit docstring

dkuegler commented:
Two things:
- Unfortunately, this PR is based in stable, which is woefully outdated. Therefore, this should be rebased onto dev (or stable after the 2.1 release).
- This PR is a merge into agirodi/stable, but it should be targeting FastSurfer/dev.

agirodi commented:
* At the time of the creation, stable wasn't outdated as far as I know. But yes it is now. I will be rebasing soon and update the helptexts accordingly.
* This PR was supposed to just be a list for myself to keep track of which files I still have to do, which is why it isn't targeting FastSurfer/dev.

agirodi committed:
docs add/edit docstrings

N4_bias_correct.py, paint_cc_into_pred.py, rewrite_mc_surface.py, rotate_sphere.py, smooth_aparc.py, spherically_project.py, spherically_project_wrapper.py

agirodi committed:
docs add/edit docstrings
